src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const ACCORDIONPANELGROUP_CLASS = 'accordionPanelGroup';
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			return this.clonePanels(accordionPanel, i, accordionPanel.props.isOpen);
+		});
+
+		this.state = {
+			clickedPanel: null
+		};
+
+		this.clonePanels = this.clonePanels.bind(this);
+		this.setClickedPanel = this.setClickedPanel.bind(this);
+	}
+
+	/**
+	 * @param clickedPanel is the `AccordionPanel` component passed in from `_handleToggle`
+	 * @returns {undefined}
+	 *
+	 * Keeps track of the clicked panel in order to support AccordionPanelGroups that only
+	 * have one panel open at a time.
+	 */
+	setClickedPanel(clickedPanel) {
```

Comment:
Thinking about this a bit more, and I'm wondering if you should just _always_ use `setState({ clickedPanel })`, but ignore the value when `multiSelectable` is `false` rather than trying to be smart about setting `this.setClickedPanel` to be false and then checking whether it's false before calling it... essentially make the method calling consistent, by putting all the logic into figuring out what to _do_ with the information it provides